plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx

```diff
@@ -70,6 +70,7 @@ const config: ControlPanelConfig = {
       controlSetRows: [
         ['groupby'],
         ['metrics'],
+        ['timeseries_limit_metric'],
```

Comment:
I will file a new PR to add orderby shared-control. we don't need this control in the non-timeseries chart.

Reply:
refer:https://github.com/apache-superset/superset-ui/pull/1112#discussion_r636270729